Makefile

```diff
@@ -36,7 +56,9 @@ all build:
 #   make check
 #   make test
 #   make check WHAT=pkg/build GOFLAGS=-v
-check test:
+check test: export STI_COVER= -cover -covermode=atomic
+check test: export STI_RACE=  -race
+check test: verify
```

Comment:
also your example is for Makefile variables, not shell variables... but you want STI_COVER to be a shell variable...right?